hypemaths/models/matrix.py

```diff
@@ -67,7 +144,18 @@ def _create_filled_matrix(dims: tuple, fill: t.Union[int, float]) -> list:
                 f"The fill value must be integer or float, but the given fill value is {type(fill)}."
             )
 
-        return [[fill] * dims[1]] * dims[0]
+        # Reason:
+        #   n = Matrix(dims=[2, 3], fill=0)
+        #   n[1][1] = 45
+        #   print(n) # Matrix([[0, 45, 0], [0, 45, 0]])
+        # to stop this from happening
```

Comment:
```suggestion
        """
        Reason:
          n = Matrix(dims=[2, 3], fill=0)
          n[1][1] = 45
          print(n) # Matrix([[0, 45, 0], [0, 45, 0]])
         to stop this from happening
         """
```
A better way for longer comments and explanations 👍🏼